core/os/memory.cpp

```diff
@@ -198,10 +229,17 @@ void Memory::free_static(void *p_ptr, bool p_pad_align) {
 		mem_usage.sub(*s);
 #endif
 
+#ifdef MIMALLOC_ENABLED
+		mi_free(mem);
+	} else {
+		mi_free(mem);
+	}
+#else
 		free(mem);
 	} else {
 		free(mem);
 	}
+#endif
```

Comment:
```suggestion
#ifdef MIMALLOC_ENABLED
		mi_free(mem);
#else
 		free(mem);
#endif
 	} else {
#ifdef MIMALLOC_ENABLED
		mi_free(mem);
#else
 		free(mem);
#endif
 	}
```
Better readability IMO